src/gm_base/geometry_files/format_last.py

```diff
@@ -144,8 +144,8 @@ def __init__(self, config={}):
         super().__init__(config)
 
     def __eq__(self, other):
-        return self.node_ids == other.node.ids \
-            and self.surface_id == other.surface_id
+        return self.node_ids == other.node_ids \
```

Comment:
Where are these comparison used? This seems to be a dangerous implementation.

Reply:
I'm not even sure if this comparison is used. I just found out that the implementation is using nonexistent attributes, so I fixed it. Also I'm not sure what is dangerous about it? It could be slow but it should work.